server/src/main/java/org/elasticsearch/action/admin/indices/alias/get/GetAliasesRequest.java

```diff
@@ -31,18 +30,24 @@ public class GetAliasesRequest extends ActionRequest implements AliasesRequest {
     private String[] indices = Strings.EMPTY_ARRAY;
     private IndicesOptions indicesOptions = DEFAULT_INDICES_OPTIONS;
 
+    @Deprecated
     public GetAliasesRequest(String... aliases) {
-        this.aliases = aliases;
-        this.originalAliases = aliases;
+        this(MasterNodeRequest.TRAPPY_IMPLICIT_DEFAULT_MASTER_NODE_TIMEOUT, aliases);
```

Comment:
It felt out of scope for this PR to refactor all these constructor callers to provide an explicit master timeout - that will have to be done in a follow-up PR.

Reply:
I would rather we did those changes first to set things up for this PR to go through without having to add back this trappy parameter.

Reply:
How would we do those changes first if there is currently - on `main` - no timeout to set in these constructors? Are you suggesting adding a constructor on `main` that takes a timeout but doesn't actually do anything with it and swapping that constructor with the ones on this branch afterward?